### PR TITLE
update release notes and highlights

### PR DESCRIPTION
**Pull request by kvalliyurnatt**
Update 2.13.0 release notes and highlights to denote that logstash is GA with 2.13.0 release

### CODE REVIEW

File: docs/release-notes/highlights-2.13.0.asciidoc [hunk 1]
```diff
@@ -21,3 +21,9 @@ ECK 2.13.0 supports managing Enterprise search resources via Helm charts, simila
 ECK 2.13.0 introduces a new option that allows a user to disable the elastic user from being created upon Elasticsearch creation. A use case for this would be when an organization/user would prefer to manage all users/roles via SSO (SAML/IDP/LDAP/etc)
 (see https://github.com/elastic/cloud-on-k8s/blob/main/docs/orchestrating-elastic-stack-applications/security/users-and-roles.asciidoc#disabling-the-default-elastic-user[documentation]).
 
+[float]
+[id="{p}-2130-eck-logstash-ga"]
+=== ECK logstash General Availability
```
Comment (pebrc) [inline docs/release-notes/highlights-2.13.0.asciidoc:26]:
```suggestion
=== Logstash on ECK general availability
```
or "generally available" not sure

File: docs/release-notes/highlights-2.13.0.asciidoc [hunk 2]
```diff
@@ -21,3 +21,9 @@ ECK 2.13.0 supports managing Enterprise search resources via Helm charts, simila
...
+
+ECK 2.13.0 moves the Logstash component to General Availability. Logstash is now considered production-ready.
```
Comment (pebrc) [inline docs/release-notes/highlights-2.13.0.asciidoc:28]:
```suggestion
With ECK 2.13.0 the support for Logstash is no longer a technical preview. Logstash managed by ECK is now considered production-ready.
```

Comment (pebrc) [reply docs/release-notes/highlights-2.13.0.asciidoc:28]: @karenzone could you maybe take a look and double check our wording?

File: docs/release-notes/highlights-2.13.0.asciidoc (hunk 1 above)
Comment (kvalliyurnatt) [reply docs/release-notes/highlights-2.13.0.asciidoc]: made it generally available

**Comment by kvalliyurnatt** [comment]
@karenzone @robbavey sorry for bumping this, we have the release next Tuesday so would like to get this in ASAP. Please take a look at the release highlights regarding logstash GA

**Review by robbavey — APPROVED** [review]
LGTM

File: docs/release-notes/highlights-2.13.0.asciidoc [hunk 3]
```diff
@@ -21,3 +21,9 @@ ECK 2.13.0 supports managing Enterprise search resources via Helm charts, simila
 ECK 2.13.0 introduces a new option that allows a user to disable the elastic user from being created upon Elasticsearch creation. A use case for this would be when an organization/user would prefer to manage all users/roles via SSO (SAML/IDP/LDAP/etc)
 (see https://github.com/elastic/cloud-on-k8s/blob/main/docs/orchestrating-elastic-stack-applications/security/users-and-roles.asciidoc#disabling-the-default-elastic-user[documentation]).
 
+[float]
+[id="{p}-2130-eck-logstash-ga"]
+=== Logstash on ECK generally available
+
+With ECK 2.13.0 the support for Logstash is no longer a technical preview. Logstash managed by ECK is now considered production-ready.
```
Comment (karenzone) [inline docs/release-notes/highlights-2.13.0.asciidoc:28]:
```suggestion
With ECK 2.13.0, support for Logstash is moving out of technical preview and is now generally available (GA). 
Logstash managed by ECK is now considered production-ready.
```

**Review by karenzone — APPROVED** [review]
Minor wording suggestion inline for your consideration.  Otherwise, LGTM! 
Thanks for getting this in.

File: docs/release-notes/2.13.0.asciidoc [hunk 4]
```diff
@@ -27,6 +28,7 @@
 === Bug fixes
 
 * Increase default memory for agent {pull}7789[#7789]
+* Correct statefulset name label for Logstash pods {pull}7788[#7788] (issue: {issue}7742[#7742])
```
Comment (naemono) [inline docs/release-notes/2.13.0.asciidoc:31]:
```suggestion
* Correct StatefulSet name label for Logstash pods {pull}7788[#7788] (issue: {issue}7742[#7742])
```

**Review by naemono — APPROVED** [review]
Minor Nit, but LGTM

**Comment by thbkrkr** [comment]
@kvalliyurnatt Maybe use this PR to regenerate the release notes following the recent backport #7811.

**Comment by kvalliyurnatt** [comment]
> @kvalliyurnatt Maybe use this PR to regenerate the release notes following the recent backport #7811.

yup already did that in the PR

**Comment by kvalliyurnatt** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|2.13|[](https://github.com/elastic/cloud-on-k8s/pull/7822)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**